docs/admin/cluster-management.md

```diff
@@ -80,6 +72,37 @@ If you want more control over the upgrading process, you may use the following w
     be created automatically when you create a new VM instance (if you're using a cloud provider that supports
     node discovery; currently this is only Google Compute Engine, not including CoreOS on Google Compute Engine using kube-register). See [Node](node.md).
 
+## Advanced Topics
+
+### Upgrading to a different API version
+When a new API is released, you may need to upgrade a cluster to support a new API version (e.g. switching from 'v1' to 'v2' when 'v2' is launched)
+
+This is an infrequent event, but it requires careful management. There is a sequence of steps to upgrade to a new API version.
+
+1. Turn on the new api version
```

Comment:
Numbers are a little mixed up. The rendering doesn't care about it, but it will bother readers. Just use `1.` for all of them, as above?